src/functional_test_runner/lib/mocha/reporter/reporter.js

```diff
@@ -130,15 +130,11 @@ export function MochaReporterProvider({ getService }) {
         '\n' +
         output
           .split('\n')
-          .slice(2) // drop the first two lines, (empty + test title)
-          .map(line => {
-            // move leading colors behind leading spaces
-            return line.replace(/^((?:\[.+m)+)(\s+)/, '$2$1');
-          })
-          .map(line => {
-            // shrink mocha's indentation
-            return line.replace(/^\s{5,5}/, ' ');
```

Comment:
Previously we were indenting all multi-line output assuming it was being prefixed, but now only output that is actually prefixed (like output from `success()`, or `warning()`) will have the subsequent lines indented. This makes the log output from methods like `log.write()` far nicer looking, but since we are basically applying out own prefix here (`fail: `) we want to keep the indentation mocha generates to make the output looking nice:

![image](https://user-images.githubusercontent.com/1329312/44416504-e4754b80-a527-11e8-910b-f7dd356b0f32.png)

Reply:
Nnnniice, thanks!